src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,12 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.List;
+
+public class BulldozerProducer
+        implements MachineProducer<Bulldozer> {
```

Comment:
No need to implement every time MachineProducer, you can implement MachineProducer via Machine.
No need to Parametrize.